api/src/main/resources/workbench.yaml

```diff
@@ -706,12 +706,16 @@ paths:
             $ref: "#/definitions/CriteriaListResponse"
 
   # Cohort Review  #######################################################################
-  /api/v1/cohortreview/participants/{cohortId}/{cdrVersionId}:
+  /api/v1/cohortreview/info/{cohortId}/{cdrVersionId}:
```

Comment:
So FWIW... when dealing with cohorts previously, I had a path like this:

https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench.yaml#L511

that encodes the workspace as well as the cohort ID.

Technically it's redundant, since the cohort ID is globally unique. But most REST APIs I've worked with still do something like this (and perhaps validate that the cohort ID you picked is actually in the workspace you picked.)

Anyway, I'm not super-attached either way, but we should probably aim for consistency... maybe either this should change or that should.